apps/mobile/firebase.json

```diff
@@ -0,0 +1,9 @@
+{
+  "react-native": {
+    "analytics_auto_collection_enabled": false,
```

Comment:
I think that we need to keep this set to false. We are going to have a prompt to enable analytics in order to be GDPR compliant.